workos/mfa.py

```diff
@@ -88,6 +89,59 @@ def enroll_factor(
 
         return WorkOSAuthenticationFactorSms.construct_from_response(response).to_dict()
 
+    def get_factor(
+        self,
+        id=None,
+    ):
+        """
+        Returns an authorization factor from it's ID.
```

Comment:
Possessive "its" does not use an apostrophe:

```suggestion
        Returns an authorization factor from its ID.
```